runtests.py

```diff
@@ -2733,9 +2735,26 @@ def runtests(options, cmd_args, coverage=None):
             refnanny = import_refnanny()
         CDEFS.append(('CYTHON_REFNANNY', '1'))
 
+    sys_version_or_limited_version = sys.version_info
     if options.limited_api:
-        CDEFS.append(('CYTHON_LIMITED_API', '1'))
-        CDEFS.append(("Py_LIMITED_API", '(PY_VERSION_HEX & 0xffff0000)'))
+        if options.limited_api is True:
+            CDEFS.append(("Py_LIMITED_API", '(PY_VERSION_HEX & 0xffff0000)'))
+        else:
```

Comment:
We do not support running the test runner in a different Python version than the tests (and I don't see a real incentive to implement that), so this seems a useless special case. Let's remove it and always generate the version string from the option value.